### DISCUSSION TITLE
stock count and countdown timer viewing location

### DISCUSSION THREAD

**Discussion by satyajittalukder** [getdokan/storegrowth-sales-booster#422] [Feature Request]
As of now the viewing location is only available for the after the product thumbnail.
In the requested feature the user must be able to change the viewing location of the modules
**this is for the single product page**
![image](https://github.com/Invizo/storegrowth-sales-booster/assets/65698588/7e6a6924-6beb-4a1b-aeb9-7d07648c5136)
 
**This is for the shop page product**
![image](https://github.com/Invizo/storegrowth-sales-booster/assets/65698588/ec9a6a15-14c7-4fe4-897b-479b039fe545)

**Comment by imtiaz-pranto** [comment]
Converting it into discussion for further work
